validation-test/SILOptimizer/lexical-lifetimes.swift

```diff
@@ -29,27 +87,71 @@ class D {
 // = Tests                                                                  {{ =
 // =============================================================================
 
-func test_localLetKeepsObjectAliveBeyondCallToClassWithWeakReference() {
+func test_localLet_keepsObjectAliveBeyondCallToClassWithWeakReference() {
   let d = D()
   let c = C(d)
-  // CHECK: cWillFoo{{.*}} test_localLetKeepsObjectAliveBeyondCallToClassWithWeakReference
+  // CHECK: cWillFoo{{.*}} test_localLet_keepsObjectAliveBeyondCallToClassWithWeakReference
   c.foo(#function)
 }
 
-func test_localVarKeepsObjectAliveBeyondCallToClassWithWeakReference() {
+func test_localVar_keepsObjectAliveBeyondCallToClassWithWeakReference() {
   var d = D()
   let c = C(d)
-  // CHECK: cWillFoo{{.*}} test_localVarKeepsObjectAliveBeyondCallToClassWithWeakReference
+  // CHECK: cWillFoo{{.*}} test_localVar_keepsObjectAliveBeyondCallToClassWithWeakReference
   c.foo(#function)
 }
 
+func test_localLet_keepsObjectAliveBeyondCallToClassWithPointer_doit(_ input: [UInt8]) {
+    let data = DataWrapper(count: input.count)
+    data.bytes.copyBytes(from: input)
+}
+func test_localLet_keepsObjectAliveBeyondCallToClassWithPointer() {
+  test_localLet_keepsObjectAliveBeyondCallToClassWithPointer_doit([1,2,3,4,50])
+}
+
+func test_localVar_keepsObjectAliveBeyondCallToClassWithPointer_doit(_ input: [UInt8]) {
+    let data = DataWrapper(count: input.count)
```

Comment:
maybe this was supposed to be `var data`?

Reply:
Thanks!  Fixed.